packages/harpguru-core/src/components/experience-mode-button/experience-mode-button.tsx

```diff
@@ -28,12 +28,19 @@ export const ExperienceModeButton = ({
   // eslint-disable-next-line no-empty-pattern
   const [] = useGlobal('activeHarpStrata')
   const [activeExperienceMode] = useGlobal('activeExperienceMode')
-  const nudgeExperienceMode = useNudgeExperienceMode()
+  // TODO: Figure out why this wasn't typesafe and try to resolve in case
```

Comment:
TODO here. Check the other action buttons and whether they have the same problem.

Reply:
I don't want to take the time to resolve this at the moment. https://github.com/js-jslog/harpguru/issues/76 raised.